corehq/util/string_utils.py

```diff
@@ -0,0 +1,7 @@
+import random
+import string
+
+
+def random_string(n=6):
```

Comment:
similar to https://github.com/dimagi/commcare-hq/blob/02681108ef907ce6f97c21c5cb415229e16985df/corehq/blobs/util.py#L50-L50